exercise-src/react-native/06-custom-components/01-solution/App.tsx

```diff
@@ -14,25 +14,23 @@ const states = [
 
 const StateList: FC = () => {
   return (
-    <View>
+    <ScrollView>
+      <View>
```

Comment:
In a future exercise, this is suddenly in the StateList component. Since that component is a full page, I think it makes sense for the title to be in it, so I normalized the location going backwards too.